editor/code_editor.cpp

```diff
@@ -1677,7 +1678,6 @@ CodeTextEditor::CodeTextEditor() {
 	error = memnew(Label);
 	scroll->add_child(error);
 	error->set_v_size_flags(SIZE_EXPAND | SIZE_SHRINK_CENTER);
-	error->add_color_override("font_color", EditorNode::get_singleton()->get_gui_base()->get_color("error_color", "Editor"));
 	error->add_font_override("font", EditorNode::get_singleton()->get_gui_base()->get_font("status_source", "EditorFonts"));
```

Comment:
Shouldn't the same be done with this one?

Reply:
This is constructor, when the theme changed this should be placed in notification method

Reply:
Ah.. you mean add_font_override ?

Reply:
Yes, like it's done currently for `text_editor`.

Reply:
Done